compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html

```diff
@@ -1,5 +1,13 @@
-<div style="height: 100%; width: 100%; display: table;">
-    <div style="display: table-cell; background-color: cornflowerblue; text-align: center; vertical-align: middle;">
-        DUMMY XML3D CANVAS!
-    </div>
+<div style="height: 100%; width: 100%; overflow: hidden;">
```

Comment:
this is still called `viewport.html`, the rest of the view is called `XML3DView`. I think we should rename this for consistency reasons.